PURL-TYPES.rst

```diff
@@ -280,6 +280,20 @@ hex
       pkg:hex/bar@1.2.3?repository_url=https://myrepo.example.com
 
 
+huggingface
+------
+``huggingface`` for Hugging Face ML models
+
+- The default repository is ``https://huggingface.co``.
+- The ``namespace`` is the model repository username or organization, if present. It is case insensitive and must be lowercased.
```

Comment:
> must be lowercased

I'm not sure it 'must' be lowercased, but otherwise the statement is correct

Reply:
The namespace on Hugging Face is case aware, but is not currently case sensitive. I would recommend removing the word 'must' since Hugging Face appears to support case.

Reply:
Doing some more testing it looks like case matters for `namespace` and `name` but not for `version`. Updated spec + tests accordingly.

Test results:

Test|Result|URL
--|--|--
Baseline|`200`|https://huggingface.co/EleutherAI/gpt-neo-1.3B/commit/797174552ae47f449ab70b684cabcb6603e5e85e
Change `namespace` case|`404`|https://huggingface.co/eleutherAI/gpt-neo-1.3B/commit/797174552ae47f449ab70b684cabcb6603e5e85e
Change `name` case|`404`|https://huggingface.co/EleutherAI/gpt-neo-1.3b/commit/797174552ae47f449ab70b684cabcb6603e5e85e
Change `version` case|`200`|https://huggingface.co/EleutherAI/gpt-neo-1.3B/commit/797174552ae47f449ab70b684cabcb6603e5e85E